ArtOfIllusion/src/artofillusion/PluginRegistry.java

```diff
@@ -203,6 +204,12 @@ private static void processJar(JarInfo jar, Map<String, JarInfo> nameMap)
         registerResource(info.type, info.id, jar.loader, info.name, info.locale);
       }
     }
+    catch(NoClassDefFoundError ncdfe)
+    {
+      new BStandardDialog("", UIUtilities.breakString(Translate.text("pluginLoadError", jar.file.getName())), BStandardDialog.ERROR).showMessageDialog(null);
```

Comment:
As previously mentioned, I think we do need to broaden this, at least to `LinkageError` if not all the way to just `Error`.

Reply:
Thanks for the update. I did some further testing with an (easy to make) broken plugin. We do need to just broaden all the way to `Error` here.

I can make the change myself, if you prefer.